apps/test/unit/storage/FirebaseStorageTest.js

```diff
@@ -602,9 +610,26 @@ describe('FirebaseStorage', () => {
         });
     });
 
+    // Some users got into a bad state where populateTables wrote storage/tables,
+    // but failed to write counters/tables due to a security rule violation. Make
+    // sure we overwrite tables for users in that state.
+    it('does overwrite existing data when counters/tables node is empty', done => {
+      const overwrite = false;
+      getDatabase().child(`storage/tables`).set(EXISTING_TABLE_DATA)
+        .then(() => {
+          FirebaseStorage.populateTable(
+            NEW_TABLE_DATA_JSON,
+            overwrite,
+            () => verifyTable(NEW_TABLE_DATA).then(done),
+            error => {throw error;});
```

Comment:
I guess it would be like saying `return throw error;`. but yeah I definitely tried that and was surprised that it didn't work.